.github/workflows/release-candidate.yaml

```diff
@@ -7,14 +7,13 @@ jobs:
   rc:
     name: Trigger release candidate build
     runs-on: ubuntu-latest
-    container: weaveworks/eksctl-build:0eb3a29941b9740f055ecb3fd9b663762a7ee9bd
+    container: weaveworks/eksctl-build:1298c3979281b6438cfce89f3215cc0001e6dfc3
     steps:
       - name: Checkout
         uses: actions/checkout@v3
         with:
           token: ${{ secrets.WEAVEWORKSBOT_TOKEN }}
           fetch-depth: 0
-          set-safe-directory: '$GITHUB_WORKSPACE'
```

Comment:
why are we removing these which were added required here https://github.com/weaveworks/eksctl/pull/5424/files

Reply:
Please see the comment I left here https://github.com/weaveworks/eksctl/pull/5402#discussion_r899230635